### ISSUE TITLE
System hang after proc_setup

### ISSUE THREAD

**Issue by kuanhsuncheng** [xvisor/xvisor#163]
Hello, I'm trying to bring up xvisor in arm64 board.

Now I found the  system would go _start_hang after "Setup Hypervisor System Control Register"

```
	/* Setup Hypervisor System Control Register */
	bl	proc_setup
	msr     sctlr_el2, x0
	dsb	sy
	isb

_start_hang:
	b	.

	.align 3
	.globl __mair_set
	.globl __tcr_set
```

Is there an exeption after `msr     sctlr_el2, x0` ?

Or Is there anything more I should look into?

Thanks.

**Comment by avpatel** [comment]
Yes, exception is expected after writing sctlr_el2 because MMU gets enabled and execution should jump to __init_vectors.

If it is jumping to _start_hang then it is not expected behaviour. Try adding "dsb sy" and "isb" before updating sctlr_el2

**Comment by kuanhsuncheng** [comment]
Hi

In the original flow (cpu_entry.S), "dsb sy" and "isb" both are added after "sctlr_el2, x0"
Is anything I can check more?

Thnaks.
